src/views/layouts/modals.blade.php

```diff
@@ -1,3 +1,78 @@
+{{-- ---------------------- Group Channel Modal ---------------------- --}}
+<div class="app-modal group-modal" data-name="addGroup">
+    <div class="app-modal-container">
+        <div class="app-modal-card" data-name="addGroup" data-modal='0'>
+            <form id="addGroupForm" action="{{ route('group-chat.create') }}" enctype="multipart/form-data" method="POST">
+                @csrf
+                <div class="app-modal-header">
+                    Create a Group Channel
+                </div>
+                <div class="app-modal-body">
+                    {{-- channel avatar --}}
+                    <div class="avatar av-l upload-avatar-preview chatify-d-flex"
+                         style="background-image: url('{{ Chatify::getUserWithAvatar(Auth::user())->avatar }}');"
+                    ></div>
+                    <p class="upload-avatar-details"></p>
+                    <label class="app-btn a-btn-primary update" style="background-color:{{$messengerColor}}">
+                        Upload New
+                        <input class="upload-avatar chatify-d-none" accept="image/*" name="avatar" type="file" />
+                    </label>
+                    {{-- End channel avatar --}}
+
+                    <div class="form-control">
+                        <label class="form-label" for="group_name">Group Name</label>
+                        <input class="form-input" type="text" name="group_name" id="group_name"
+                               placeholder="Enter a name"
+                               required="required"
+                        />
+                    </div>
+                    <div class="form-control">
+                        <label class="form-label" for="search">Select user(s) to the group</label>
+                        <input class="form-input user-search" type="text" name="search" placeholder="Search"/>
```

Comment:
I suggest to make this field `autocomplete="off"` I had a list of previously looked searches. (Linux newest Firefox).